mediacenter/tests/test_managers.py

```diff
@@ -0,0 +1,47 @@
+import pytest
+
+from ..models import Document
+
+pytestmark = pytest.mark.django_db
+
+
+def test_video_should_return_only_video(video, image, audio):
```

Comment:
why not add pdf and text here and test on them too ? (same for all tests in this file, using all doc types)